js/renovation/ui/__tests__/check_box.test.tsx

```diff
@@ -638,10 +638,10 @@ describe('CheckBox', () => {
       afterEach(() => jest.resetAllMocks());
 
       describe('useInkRiple', () => {
-        it('should be true if material theme', () => {
+        it('should be true (temporaly replaced with false) if material theme', () => {
```

Comment:
we should just skip this test with no any other changes and add comment-note at the test begining